modules/dedicated-managing-dedicated-administrators.adoc

```diff
@@ -11,10 +11,10 @@ link:https://console.redhat.com/openshift[{cloud-redhat-com}] site.
 
 [id="dedicated-administrators-adding-user_{context}"]
 == Adding a user
-. Navigate to the *Cluster Details* page and *Users* tab.
```

Comment:
Need to verify that this is correct for 4.10. In appears to have been changed in: https://github.com/openshift/openshift-docs/pull/27872/files.

Reply:
Leaving comments in place. These files appear to be unused, but I'm going to continue with the CP for consistency with main.